corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -445,6 +456,17 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
+        updateCheckboxes: function () {
+            if (this.isMultiSelect) {
+                this.children.each(function (childView) {
+                    if (childView.parentView.selectedCaseIds.indexOf(childView.model.id) !== -1) {
```

Comment:
My tendency would be to directly access the parent view since you're already in it, by adding `var self = this;` before the `each` and then using `self. selectedCaseIds` here (or maybe `this` is still the parent view inside the each? I'm not sure.). But I can't think of an argument why that's better, so let's call it a stylistic difference.

Reply:
I like this better and also implemented it in a different PR too so it's will be consistent [7cd6bd0](https://github.com/dimagi/commcare-hq/pull/31478/commits/7cd6bd047e7ad83c8dcb170403e5377d74180e96)